src/Sylius/Component/Core/spec/Model/ProductVariantSpec.php

```diff
@@ -51,6 +56,11 @@ function its_price_should_be_mutable()
         $this->setPrice(499)->getPrice()->shouldReturn(499);
     }
 
+    function its_original_price_should_be_mutable()
+    {
+        $this->setOriginalPrice(399)->getOriginalPrice()->shouldReturn(399);
```

Comment:
According to my previous comment, this should be split to ``$this->setOriginalPrice(399);`` and ``$this->getOriginalPrice()->shouldReturn(399);``.